### DISCUSSION TITLE
请问大模型如何启动

### DISCUSSION THREAD

**Discussion by steph730** [RapidAI/RapidRAG#16] [Q&A] [answered]
是在config.yaml将llm-api后的内容改成本地大模型存储路径嘛

**Comment by SWHL** [comment, marked as answer]
大模型启动是和当前项目分开的。
`config.yaml`中`llm_api`指的是 在本地部署大模型之后的接口路径，不是大模型存储路径。
大模型启动可以参考[ChatGLM-6B](https://github.com/THUDM/ChatGLM-6B/blob/main/api.py)来单独启动
启动之后，将接口路径 赋值给 `llm_api`
